.github/workflows/autobuild-dependabot-assets.yml

```diff
@@ -0,0 +1,41 @@
+name: Add built assets to Dependabot PRs
+
+on:
+  pull_request_target:
+    branches:
+      - dev
+    paths:
+      - package.json
+      - package-lock.json
+
+jobs:
+  autobuild:
+    if: startsWith(github.head_ref, 'autobuild') == false && startsWith(github.head_ref, 'dependabot') == true && github.event.pull_request.head.repo.full_name == github.repository
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out Dependabot branch
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+      - name: Install dependencies
+        run: npm ci
+      - name: Build assets
+        run: npm run build
+      - name: Create pull request with built assets
+        id: cpr
+        uses: peter-evans/create-pull-request@v4
+        with:
+          commit-message: "chore(build): build assets"
+          title: "chore(build): build assets"
+          body: This is an auto-generated PR to build assets changed by Dependabot.
+          labels: dependencies, automerge
+          branch: "autobuild/${{ github.head_ref }}"
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@greatislander Will we need to define/add a GITHUB_TOKEN secret as an organizational or repository secret for this to work? If so, we already have an organizationl secret defined that is used for similar things. I believe it's called `PAT_FOR_GITHUB_ACTIONS`. Could/should we change token reference to match that name?